transformations/lost_in_translation/transformation.py

```diff
@@ -0,0 +1,110 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import random
+from transformers import pipeline
+
+class LostInTranslation(SentenceOperation):
+  '''
+  Repeated translation of input text
+  '''
+  tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+  languages = ["en"]
+  heavy = True
```

Comment:
you can remove this as of now so that the build can be tested. I will add it later.